engine/ConsoleRunnerDotNet6/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using JustDecompileCmdShell;
```

Comment:
All new files should have AGPL license notice on top. You can take the text from other files.